res/main.qml

```diff
@@ -0,0 +1,36 @@
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+import Mixxx 1.0
+
+ApplicationWindow {
+    id: window
+    width: 34
+    height: 100
+    visible: true
+
+    Slider {
+        id: slider
```

Comment:
```suggestion
        id: slider
        orientation: Qt.Vertical
```

Reply:
Ah, I think I tried `Qt.vertical` before instead of `Qt.Vertical` which is why it didn't work.

Reply:
is there no error output from the patser to catch those typos, like the qss parser complains about unknown properties?

Reply:
Maybe, but if there is, I missed it in the sea of logging we have.